scripts/accounts/gcloud/setup-firebase.sh

```diff
@@ -46,6 +46,14 @@ checkMandatoryArguments() {
         echo -ne "${white}" >&2
         exit 2
     fi
+    # Firestore region
+    if [ -z "$firestoreRegion" ];
+    then
+        echo -e "${red}Error: Missing paramenters, -r is mandatory." >&2
```

Comment:
```suggestion
        echo -e "${red}Error: Missing parameters, -r is mandatory." >&2
```

Fix other occurrences.

Reply:
@capf4bbo Other occurrences of mistyped parameters are not fixed yet.